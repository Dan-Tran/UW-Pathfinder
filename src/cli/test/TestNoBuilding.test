# Verify that the program correctly handles buildings that don't exist
# Destination does not exist
r
CSE
DAN

# Source does not exist
r
DAN
CSE

# Both does not exist
r
DAN
DDD

q