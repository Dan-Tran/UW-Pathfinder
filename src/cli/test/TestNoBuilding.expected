Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program correctly handles buildings that don't exist
# Destination does not exist
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: DAN

Enter an option ('m' to see the menu):
# Source does not exist
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: DAN

Enter an option ('m' to see the menu):
# Both does not exist
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: DAN
Unknown building: DDD

Enter an option ('m' to see the menu):
